pages/en/glossary.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Glossary
+---
+
+- **The Graph**: A decentralized protocol for indexing and querying data.
+
+- **Query**: A request for data. In the case of The Graph, a query is a request for data from a subgraph that will be answered by an Indexer.
+
+- **GraphQL**: A query language for APIs and a runtime for fulfilling those queries with your existing data. The Graph uses GraphQL to query subgraphs.
+
+- **Subgraph**: A custom API built on blockchain data that can be queried using the [GraphQL](https://graphql.org/). Developers can build, deploy and publish subgraphs to The Graph's decentralized network. Then, Indexers can begin indexing subgraphs to make them available to be queried by subgraph consumers.
+
+- **Hosted Service**: A temporary scaffold service for building and querying subgraphs while The Graph's decentralized network matures the cost of service, quality of service, and developer experience.
+
+- **Indexers**: Network participants that run indexing nodes to serve queries to subgraphs.
+
+- **Indexer revenue streams**: Indexer's are rewarded in GRT with two components: indexing rewards and query fee rebates.
+
+  1. **Query fee rebates**: Payments from subgraph consumers for serving queries on the network.
+
+  2. **Indexing rewards**: The rewards that Indexers receive for indexing subgraphs. Indexing rewards are generated via new issuance of 3% GRT annually.
+
+- **Indexer's self stake**: The amount of GRT that Indexers stake to process subgraphs on the network. The minimum is 100,000 GRT, and there is no upper limit.
+
+- **Delegators**: Individuals who own GRT and stake (delegate) their GRT to Indexers. This allows Indexers to increase their stake in subgraphs on the network. In return, Delegators receive a portion of the Indexing Rewards that Indexers receive for processing subgraphs.
```

Comment:
The definition of Indexers started with "Network participants that...", and I think it might be good to repeat that for Delegators and Curators.

Reply:
Good call! Fixed all roles accordingly! Thank you!